src/ext/coms.c

```diff
@@ -633,12 +634,23 @@ static void _dd_curl_set_connect_timeout(CURL *curl) {
 }
 
 static void _dd_curl_set_hostname(CURL *curl) {
+    char *url = get_dd_trace_agent_url();
+    if (url && url[0]) {
+        size_t agent_url_len = strlen(url) + sizeof(TRACE_PATH_STR);
+        char *agent_url = malloc(agent_url_len);
+        sprintf(agent_url, "%s%s", url, TRACE_PATH_STR);
+        curl_easy_setopt(curl, CURLOPT_URL, agent_url);
+        free(url);
+        free(agent_url);
+        return;
+    }
+
     char *hostname = get_dd_agent_host();
     int64_t port = get_dd_trace_agent_port();
     if (port <= 0 || port > 65535) {
         port = 8126;
     }
-
+    
```

Comment:
The linter doesn't like the extra whitespace on this line.

Reply:
gotcha